pkg/cmd/flags/event.go

```diff
@@ -0,0 +1,119 @@
+package flags
+
+import (
+	"strings"
+
+	"github.com/aquasecurity/tracee/pkg/errfmt"
+)
+
+// PolicyEventMap maps policy id to its pre-parsed event flag fields
+type PolicyEventMap map[int]policyEvents
+
+// policyEvents holds pre-parsed event flag fields of one policy
+type policyEvents struct {
+	policyName string
+	eventFlags []*eventFlag
```

Comment:
Why a pointer?

Reply:
This was a continuation of the initial effort. But I can assume that returning a slice of values instead of a slice of pointers would be better in this case, since it's a slice which is referenced by a pointer, so it won't make new copies of eventFlag values on returning. Will change it to `[]eventFlag`.

Reply:
Changed also `[]*scopeFlag` to `[]scopeFlag`.